Splash/Splash/ShowersViewController.m

```diff
@@ -44,14 +57,23 @@ -(void) getData {
           for (int i = objects.count - 1; i >= 0; i = i - 1) {
               [self.showersArray addObject:objects[i]];
           }
-          //self.showersArray = objects;
+          self.filteredData = self.showersArray;
           [self.showersTableView reloadData];
+          //self.showersArray = objects;
       } else {
         // Log details of the failure
         NSLog(@"Error: %@ %@", error, [error userInfo]);
       }
     }];
     [self.refreshControl endRefreshing];
+    
+    
+}
+
+- (void) dismissKeyboard
+{
+    //Code to handle the gesture
+    [self.searchBar resignFirstResponder];
 }
 
 - (UIView *)tableView:(UITableView *)tableView viewForHeaderInSection:(NSInteger)section {
```

Comment:
consider using #pragma mark to properly structure your code. for eg: table view datasource, helper methods, etc